mautic/mautic
# Set user in token storage during import process

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Y
| New feature/enhancement? (use the a.x branch)      | N
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | Y 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR sets the active user in token storage to the user who created the import to prevent the "You do not have permission to create Items for Custom Object 1" error.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a custom object with only the name custom field
3. Navigate to the list view for that custom object then create an import using the attached CSV
4. Process the import, you'll have nothing imported and have a ton of errors about permissions.
